trunk/src/app/srs_app_gb28181.hpp

```diff
@@ -187,6 +215,7 @@ class SrsGbListener : public ISrsListener, public ISrsTcpHandler
 public:
     srs_error_t initialize(SrsConfDirective* conf);
     srs_error_t listen();
+    srs_error_t listen_api();
```

Comment:
is the `private` access scope much suitable for `listen_api`?

Reply:
Very attentive, thank you